201-vmss-master-slave-customscript/azuredeploy.parameters.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
remove parameters that have defaultValue, use GEN values for all others

Reply:
updated

Reply:
still needed